.github/workflows/release.yaml

```diff
@@ -0,0 +1,85 @@
+name: release
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  IMAGE_REGISTRY: ghcr.io
+  IMAGE_REPO: ${{ github.repository_owner }}/golden-container
+  IMAGE_TAGS: latest
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    permissions:
+      # Needed for signing the container image with GitHub OIDC Token
+      id-token: write
+      # Needed to push to ghcr.io registry
+      packages: write
+
+    outputs:
+      image:  ${{ env.IMAGE_REGISTRY }}/${{ env.IMAGE_REPO }}
+      digest: ${{ steps.push-image.outputs.digest }}
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install Cosign
+      uses: sigstore/cosign-installer@main
```

Comment:
Let's pin all of the actions. Like [this](https://github.com/enterprise-contract/golden-container/blob/1dd33c062331e68c105085af1f4acf70cbc839de/.github/workflows/scorecards.yml#L34).